libraries/fftw-mpi/README

```diff
@@ -0,0 +1,26 @@
+FFTW (Fastest Fourier Transform in the West) is a free collection of
```

Comment:
Not sure if this is a good idea to this package willy. I will delegate to your opinion here.